mne/source_estimate.py

```diff
@@ -378,9 +378,10 @@ def _get_src_type(src, vertices):
 
 
 def _make_stc(data, vertices, src=None, tmin=None, tstep=None, subject=None,
-              vector=False, source_nn=None):
+              vector=False, source_nn=None, src_type=None):
```

Comment:
`src_type` should replace the `src` parameter. We don't need both, right, since the only thing `src` is used for is a call to `_get_src_type`.

Reply:
Not that many:

```python
beamformer/_lcmv.py
17:from ..source_estimate import _make_stc, SourceEstimate
349:        yield _make_stc(sol, vertices=filters['vertices'], tmin=tmin,

beamformer/_dics.py
16:from ..source_estimate import _make_stc
371:            stcs.append(_make_stc(sol, vertices=filters['vertices'], tmin=tmin,
570:    return (_make_stc(source_power.reshape(-1, n_freqs), vertices=vertices,
657:        yield _make_stc(sol, vertices=vertno, tmin=tmin, tstep=tstep,
971:        stc = _make_stc(sol_final[i_freq, :, :].T, vertices=stc.vertices,

simulation/tests/test_raw.py
49:def _make_stc(raw, src):
78:    stc = _make_stc(raw, src)
338:    stc = _make_stc(raw, src)

minimum_norm/time_frequency.py
10:from ..source_estimate import _make_stc
164:        stc = _make_stc(power, vertices=vertno, tmin=tmin, tstep=tstep,
517:    stc = _make_stc(psd, vertices=vertno, tmin=fmin * 1e-3,
613:        stc = _make_stc(psd, tmin=fmin, tstep=fstep, vertices=vertno,

minimum_norm/inverse.py
35:from ..source_estimate import _make_stc
949:    stc = _make_stc(sol, vertno, tmin=tmin, tstep=tstep, subject=subject,
1088:    stc = _make_stc(sol, vertno, tmin=tmin, tstep=tstep, subject=subject,
1155:        stc = _make_stc(sol, vertno, tmin=tmin, tstep=tstep, subject=subject,

Reply:
and only very few of them currently use the `src` parameter.